src/Service/OrbitService.cpp

```diff
@@ -110,13 +110,13 @@ bool IsSshConnectionAlive(std::chrono::time_point<std::chrono::steady_clock> las
              .count() < timeout_in_seconds;
 }
 
-std::unique_ptr<OrbitGrpcServer> CreateGrpcServer(uint16_t grpc_port, bool dev_mode) {
+ErrorMessageOr<std::unique_ptr<OrbitGrpcServer>> CreateGrpcServer(uint16_t grpc_port,
+                                                                  bool dev_mode) {
   std::string grpc_address = absl::StrFormat("127.0.0.1:%d", grpc_port);
   ORBIT_LOG("Starting gRPC server at %s", grpc_address);
   std::unique_ptr<OrbitGrpcServer> grpc_server = OrbitGrpcServer::Create(grpc_address, dev_mode);
   if (grpc_server == nullptr) {
-    ORBIT_ERROR("Unable to start gRPC server");
-    return nullptr;
+    return ErrorMessage{"Unable to start gRPC server"};
```

Comment:
There are a couple of error messages like this one that previously were only logged. But now I replaced them by returning `ErrorMessage`. That means they will be shown to the user in the UI's error dialog. Let me know if you think that's a problem.

Reply:
I'm not very sure about showing this to our user. Does the user know what the "grpc server" means to them?

Reply:
We have to assume that the user doesn't understand the error message. The question is whether this is a problem. Of course it is not displayed on its own. The message will start with explaining that Orbit failed to start the service and this error message will be given as a more detailed explanation.

Reply:
I see. I missed the fact that the error message start with information that Orbit failed to start the service. Thanks for the explanation!

Reply:
So what's your opinion here? Could this still be confusing? Ideally I'd like to have an error dialog that only shows the general message and the user can click on a "Details" button to get this part of the error message. But unfortunately we don't have the infrastructure to do that.

Reply:
I think it should be fine as the error message starts with indicating that this is an error for starting the service.

Reply:
I think it's fine and good, a user could tell us more about an error here without having to look at the logs.

Reply:
This was basically also my line of reasoning. Thank you to both of you for your input.